### ISSUE TITLE
Gather a list of address data that's too large to upload

### ISSUE THREAD

**Issue by symroe** [openregister/address-matching-data#4]
e.g. https://alpha.openaddressesuk.org/developers/download

**Comment by symroe** [comment]
(asking for this as I can't access the wiki from the outside – maybe it should be a checked in file?)

**Comment by psd** [comment]
Could write a Makefile to download and turn the data into a bulk testcases file.

**Comment by symroe** [comment]
Is it really useful to have that much data for a test case?

**Comment by psd** [comment]
Good point, which we should explain as we get more confident about what we're doing ;)

We're seeing value in finding as many real addresses as we can as a way of benchmarking matching algorithms.

Bulk addresses are also useful for finding testcases, which should be minimal to describe the edges of address matching.

 I'm not so sure about the openaddressing addresses as they're constructed data, so probably not a great source of variance!

**Comment by frabcus** [comment]
Turns out that yes, bulk data is useful for tests cases. Can do large scale matching without postcode, then use postcode as a checksum as to which were right or wrong, then use that for training lots of algorithms.

If Open Addresses has lots of hand coded addresses it might be useful.
